recipe/0068-newspaper/newspaper_issue_1-manifest.json

```diff
@@ -0,0 +1,176 @@
+{
+  "@context": [
+    "http://iiif.io/api/presentation/3/context.json"
+  ],
+  "type": "Manifest",
+  "id": "{{ id.url }}",
+  "partOf": [
+    {
+      "id": "{{ id.path }}/newspaper_title-collection.json",
+      "type": "Collection"
+    }
+  ],
+  "label": {
+    "de": [
+      "Berliner Tageblatt - 1925-02-16"
+    ]
+  },
+  "metadata": [
+    {
+      "label": {
+        "en": [
+          "type"
+        ]
+      },
+      "value": {
+        "none": [
+          "http://schema.org/PublicationIssue",
+          "http://data.europeana.eu/concept/base/18"
+        ],
+        "en": [
+          "Analytic serial",
+          "Newspaper",
+          "Newspaper Issue"
+        ]
+      }
+    },
+    {
+      "label": {
+        "en": [
+          "language"
+        ]
+      },
+      "value": {
+        "none": "de",
+        "en": [
+          "German"
+        ]
+      }
+    }
+  ],
+  "thumbnail": [
+    {
+      "id": "https://api.europeana.eu/api/v2/thumbnail-by-url.json?uri=https%3A%2F%2Fiiif.europeana.eu%2Fimage%2F2YMIN6YXMQ6COVM5AO2XKB5KMCKPMT2YKEKNMAGHVRBIHOOY4AVA%2Fpresentation_images%2F9340afd0-ffe2-11e5-b68d-fa163e60dd72%2Fnode-2%2Fimage%2FSBB%2FBerliner_Tageblatt%2F1925%2F02%2F16%2F0%2FF_SBB_00001_19250216_054_079_0_001%2Ffull%2Ffull%2F0%2Fdefault.jpg&type=TEXT",
+      "type": "Image",
+      "format": "image/jpeg"
+    }
+  ],
+  "navDate": "1925-02-16T00:00:00Z",
+  "rights": "http://creativecommons.org/publicdomain/mark/1.0/",
+  "requiredStatement": {
+    "label": {
+      "en": [
+        "Attribution"
+      ]
+    },
+    "value": {
+      "en": [
+        "Berliner Tageblatt - https://www.europeana.eu/portal/record/9200355/BibliographicResource_3000096302605.html. Staatsbibliothek zu Berlin - Preußischer Kulturbesitz. Public Domain Mark - http://creativecommons.org/publicdomain/mark/1.0/"
+      ]
+    }
+  },
+  "logo": [
+    {
+      "id": "https://style.europeana.eu/images/europeana-logo-default.png"
+    }
+  ],
+  "seeAlso": [
+    {
+      "id": "https://www.europeana.eu/api/v2/record/9200355/BibliographicResource_3000096302513.json-ld",
+      "format": "application/ld+json",
+      "profile": "http://www.europeana.eu/schemas/edm/"
+    }
+  ],
+  "start": {
+    "id": "https://iiif.europeana.eu/presentation/9200355/BibliographicResource_3000096302513/canvas/p1",
+    "type": "Canvas"
+  },
```

Comment:
Newspaper group review: can we remove this as we don't believe its necessary as the first canvas should be shown and it adds complexity. We though this would be useful in a recipe for Newspaper articles

Reply:
done